### ISSUE TITLE
Seems like aria-label is missing in the iframe. Is fixing this in the backlog?

### ISSUE THREAD

**Issue by abhishek-msft** [howlowck/react-powerbi#39]
![image](https://user-images.githubusercontent.com/66029786/113548990-33574d00-960e-11eb-8515-8062899d0775.png)
